### PR TITLE
Adds OpenSearch + Bottlerocket + Fluent Bit+ k8s  blog post

### LINKED ISSUE #855: How to use fluentbit and OpenSearch with Bottlerocket logs
**Describe the blog post your would like to write**
This blog post will describe how to use Fluentbit to gather host container logs from Bottlerocket OS and then put them into OpenSearch to search and analyze.

**What is the title of the blog post?**
How to use fluentbit and OpenSearch with Bottlerocket logs

**Who are the authors?**
Kyle Davis

**What is the proposed posting date?**
July 15-ish

### PR DESCRIPTION
**Pull request by stockholmux**
### Description
Adds a blog post, changes bio

### Issues Resolved
#855

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by stockholmux** [comment]
Looks like there is old merge commits that are in here, but only 3 files changed (what maters).

**Comment by stockholmux** [comment]
Also, it's set for July 15 (Friday) post date, but I'm flexible - feel free to fit it into the calendar.

File: _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown [hunk 1]
```diff
@@ -0,0 +1,401 @@
+---
+layout: post
+title:  "How to use Fluent Bit and OpenSearch with Bottlerocket and Kubelet logs"
```
Comment (dtaivpp) [inline _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown:3]: Could we shorten this to 'Using Fluent Bit...', just seems like a really long title.

File: _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+categories:
+  - technical-post
+excerpt: "Today, I’m going to show you how to run OpenSearch with the OpenSearch Operator on Kubernetes using the Bottlerocket then add Fluent Bit to collect logs from the nodes of the same Kubernetes cluster."
+---
+
+This place seems familiar 👀
+
+Today, I’m going to show you how to run OpenSearch with the OpenSearch Operator on Kubernetes using the Bottlerocket then add Fluent Bit to collect logs from the nodes of the same Kubernetes cluster. Phew. If that seems like a lot, don’t worry - this post will take it bit-by-bit. I’ll assume you know what OpenSearch is, but what about the rest of that stuff?
+
+* **OpenSearch Operator for Kubernetes** controls and abstracts configuration of OpenSearch & OpenSearch Dashboards in the Kubernetes environment.
```
Comment (dtaivpp) [inline _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown:16]: Can we link back to each of these projects in their heading?

**Review by dtaivpp — COMMENTED** [review]
Other than those two things LGTM

File: _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown (hunk 1 above)
Comment (stockholmux) [reply _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown:3]: I like it.

File: _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown (hunk 2 above)
Comment (stockholmux) [reply _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown:16]: 10-4, will do.

**Review by natebower — COMMENTED** [review]
@stockholmux @krisfreedain Please see my comments and changes, and let me know if you have any questions. Thanks!

File: _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+```
+
+Why are there five different `fluent-bit-xxxx` pods? In my case I’m running a cluster of five nodes, the operator knows the size of the cluster and has deployed one instance per node.
+
+
+## Bringing it all together
+
+OK, now that you have all the components, you can start seeing how this all comes together. Fire up OpenSearch Dashboards and head over to Dev Tools (Hamburger menu > Management heading).
+
+On the left side of the screen you can type your adhoc queries. First, run the following query to confirm that you’ve go a Fluent Bit sourced index:
```
[the remark this answers is not in the document]
Comment (stockholmux) [reply _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown:378]: @natebower how about if I just drop the word `adhoc` in this context?

[the remark this answers is not in the document]
Comment (natebower) [reply _posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown:378]: @stockholmux That works.

**Comment by stockholmux** [comment]
@krisfreedain I'll have the changes all made by tomorrow morning - what is the new publish date?

**Comment by krisfreedain** [comment]
@stockholmux I will be able to publish it tomorrow

**Review by krisfreedain — APPROVED** [review]
🔥 great to have a new post from you @stockholmux
